app/views/handlers/feedback.py

```diff
@@ -0,0 +1,102 @@
+from functools import cached_property
+
+from flask_babel import gettext
+
+from app.forms.questionnaire_form import generate_form
+from app.globals import get_session_store
+from app.views.contexts.feedback_form_context import build_feedback_context
+
+
+class FeedbackNotEnabled(Exception):
+    pass
+
+
+class FeedbackAlreadySent(Exception):
+    pass
+
+
+class Feedback:
+    PAGE_TITLE = gettext("Feedback")
+    QUESTION_SCHEMA = {
+        "type": "General",
+        "id": "feedback",
+        "title": "Give feedback about this service",
+        "label": "Select what feedback is about",
+        "answers": [
+            {
+                "type": "Radio",
+                "id": "feedback-type",
+                "mandatory": True,
+                "options": [
+                    {
+                        "label": "The census questions",
+                        "value": "The census questions",
+                        "description": "For example, question not clear, answer option not relevant",
+                    },
+                    {
+                        "label": "Page design and structure",
+                        "value": "Page design and structure",
+                    },
+                    {
+                        "label": "General feedback about this service",
+                        "value": "General feedback about this service",
+                    },
+                ],
+                "validation": {
+                    "messages": {
+                        "MANDATORY_RADIO": "Select what your feedback is about"
+                    }
+                },
+            },
+            {
+                "id": "feedback-text",
+                "label": "Enter your comments",
+                "description": "For example, question not clear, answer option not relevant",
```

Comment:
On the design in the epic, this is:
```
                "label": "Enter your feedback",
                "description": "Do not include confidential information, such as your contact details.",
```
Worth confirming what it should be.

Reply:
Yours is right, looks like I copied and pasted the description from one of the radio boxes